village/src/components/SmurfForm.js

```diff
@@ -13,6 +14,20 @@ class SmurfForm extends Component {
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
+    const newSmurf = {
+      name: this.state.name,
+      age: this.state.age,
+      height: this.state.height,
+    }
+    axios
+    .post("http://localhost:3333/smurfs", newSmurf)
+    .then(response => {
+      this.setState({ smurfs: response.data });
+      this.props.history.push('/');
+      window.location.reload();
```

Comment:
If you can get App to update it's state properly, you won't need to use window.location.reload, which isn't ideal for client-side app